__tests__/welcome.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import App from '../src/components/App';
+import {Welcome} from '../src/components/Welcome';
+import {props} from '../__mockData__/redux-mock-data'
+describe('Render root app component', () => {
+    it('should render the root welcome component successfully', () => {
+        const wrapper = mount(<Welcome  {...props}/>);
+        expect(wrapper.text()).toEqual('Hello world To barefoot nomadGreetingGoodBye');   
```

Comment:
This isn't a proper test, the page needs to be tested properly e.g `expect(wrapper.find('h1').text()).toEqual('welcome To barefoot nomad');` `expect(wrapper.find('h1').length).toBe(1);` and so on.